plugins/steelseries/fu-steelseries-fizz.c

```diff
@@ -661,19 +661,19 @@ fu_steelseries_fizz_setup(FuDevice *device, GError **error)
 	g_autofree gchar *version = NULL;
 
 	/* in bootloader mode */
-	if (fu_device_has_private_flag(device, FWUPD_DEVICE_FLAG_IS_BOOTLOADER))
+	if (fu_device_has_flag(device, FWUPD_DEVICE_FLAG_IS_BOOTLOADER))
 		return TRUE;
 
 	/* FuUsbDevice->setup */
 	if (!FU_DEVICE_CLASS(fu_steelseries_fizz_parent_class)->setup(device, error))
 		return FALSE;
 
 	/* skip if in bootloader mode */
-	if (fu_device_has_private_flag(device, FWUPD_DEVICE_FLAG_IS_BOOTLOADER))
+	if (fu_device_has_flag(device, FWUPD_DEVICE_FLAG_IS_BOOTLOADER))
 		return TRUE;
 
 	/* it is a USB receiver */
-	if (fu_device_has_private_flag(device, FU_STEELSERIES_DEVICE_FLAG_IS_RECEIVER)) {
+	if (fu_device_has_flag(device, FU_STEELSERIES_DEVICE_FLAG_IS_RECEIVER)) {
```

Comment:
ohh this one is a private flag, it starts with `FU_STEELSERIES_DEVICE_` -- good catch on the others tho

Reply:
:facepalm: